ale_linters/javascript/flow.vim

```diff
@@ -21,11 +21,15 @@ endfunction
 
 function! ale_linters#javascript#flow#GetCommand(buffer) abort
   return ale_linters#javascript#flow#GetExecutable(a:buffer)
-  \   . ' check-contents --respect-pragma --json --from ale'
+  \   . ' check-contents --respect-pragma --json --from ale %s'
```

Comment:
Ahh, good to know! That makes matters much easier than. This behaviour is similar to the behaviour of `--stdin-filename` for `eslint`.